kola/tests/etcd/discovery.go

```diff
@@ -23,12 +23,12 @@ import (
 	"github.com/coreos/mantle/platform"
 )
 
-func DiscoveryV2(cluster platform.Cluster) error {
-	return discovery(cluster, 2)
+func DiscoveryV2(c platform.Cluster, _ func(string, platform.Machine) error) error {
```

Comment:
How about define that func as a type to minimize the copy/pasting of this signature all over the place.